Adapted from this upstream git commit:  2a80f8117f40dc6a3fce30c6d0f15f3c5af55ccf
http://git.infodrom.org/?p=infodrom/sysklogd;a=commit;h=2a80f8117f40dc6a3fce30c6d0f15f3c5af55ccf

diff -Nur sysklogd-1.5.orig//syslogd.c sysklogd-1.5/syslogd.c
--- sysklogd-1.5.orig//syslogd.c	2007-07-04 14:04:01.000000000 -0500
+++ sysklogd-1.5/syslogd.c	2010-10-14 13:12:15.808000011 -0500
@@ -2392,7 +2392,7 @@
 		if (*p == '\0' || *p == '#')
 			continue;
 #if CONT_LINE
-		strcpy(cline, p);
+		memmove(cline, p, strlen(p)+1);
 #endif
 		for (p = strchr(cline, '\0'); isspace(*--p););
 #if CONT_LINE
